pkgs/tools/graphics/enblend-enfuse/default.nix

```diff
@@ -1,26 +1,39 @@
-{ lib, stdenv, fetchurl
-, boost, freeglut, glew, gsl, lcms2, libpng, libtiff, libGLU, libGL, vigra
-, help2man, pkg-config, perl, texlive }:
+{ lib, stdenv, fetchzip
+, autoreconfHook
+, boost
+, freeglut
+, glew
+, gsl
+, lcms2
+, libpng
+, libtiff
+, libGLU
+, libGL
+, vigra
+, help2man
+, pkg-config
+, perl
+, texlive
+}:
 
 stdenv.mkDerivation rec {
   pname = "enblend-enfuse";
-  version = "4.2";
+  version = "unstable-2022-03-06";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/enblend/${pname}-${version}.tar.gz";
-    sha256 = "0j5x011ilalb47ssah50ag0a4phgh1b0wdgxdbbp1gcyjcjf60w7";
+  src = fetchzip {
+    url = "https://sourceforge.net/code-snapshots/hg/e/en/enblend/code/enblend-code-0f423c72e51872698fe2985ca3bd453961ffe4e0.zip";
```

Comment:
This is a 404 for me.

Reply:
Weird, commit is still on the tree: https://sourceforge.net/p/enblend/code/ci/0f423c72e51872698fe2985ca3bd453961ffe4e0/tree/

Reply:
Clicked on "generate snapshot" or something like that and regenerated the same URL: https://sourceforge.net/code-snapshots/hg/e/en/enblend/code/enblend-code-0f423c72e51872698fe2985ca3bd453961ffe4e0.zip

Reply:
¿Could you retry?

Reply:
Yeah I went to investigate and ended up at https://sourceforge.net/p/enblend/code/ci/default/tarball, which linked to the same URL.

It seems to work now... hopefully just some temporary stupidity from SF.